src/index.js

```diff
@@ -11,6 +11,7 @@ const port = process.env.PORT || 3000;
 const app = express();
 
 app.use(morgan("dev"));
+app.use(bodyParser.urlencoded({ extended: false }));
```

Comment:
Can you please this line? What does it do? Why do we need it?
I didn't really understand it from the npm page.

Reply:
https://stackoverflow.com/questions/38306569/what-does-body-parser-do-with-express

That's the best explanation I could find. We need bodyParser to read through the form input and store it as a json. Without this line the req.body in blogs.js line 16 doesn't work and no data base entry is created.